LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift

```diff
@@ -316,14 +325,29 @@ private struct CorrectionRangeOverridesGuardrailWarning: View {
     private func singularWarningTitle(for threshold: SafetyClassification.Threshold) -> Text {
         switch threshold {
         case .minimum, .belowRecommended:
-            return Text("Low Correction Value", comment: "Title text for the low correction value warning")
+            switch preset {
+            case .preMeal:
+                return Text("Low Pre-meal Value", comment: "Title text for the low pre-meal value warning")
+            case .workout:
+                return Text("Low Workout Value", comment: "Title text for the low workout value warning")
+            }
         case .aboveRecommended, .maximum:
-            return Text("High Correction Value", comment: "Title text for the high correction value warning")
+            switch preset {
+            case .preMeal:
+                return Text("High Pre-Meal Value", comment: "Title text for the low pre-meal value warning")
+            case .workout:
+                return Text("High Workout Value", comment: "Title text for the high workout value warning")
+            }
         }
     }
 
     private var multipleWarningTitle: Text {
-        Text("Correction Values", comment: "Title text for multi-value correction value warning")
+        switch preset {
```

Comment:
Hm.  I haven't thought hard about this, but, isn't it possible that a `multipleWarningTitle` might involve more than one preset?  I didn't expect this to be part of the change...

Reply:
No, it is only ever 1 preset. Hence why I was comfortable using the `crossedThresholds` originally.

Reply:
I was only expecting a change to `singleWarningTitle`.  So...it is not possible to have a multiple warning that includes both `preMeal` and `workout`?  `crossedThresholds` is a dictionary of `[CorrectionRangeOverrides.Preset: [SafetyClassification.Threshold]]`, so I'm not sure how this would work in that case.

Reply:
Not sure why the dictionary is there, to be honest. I'm happy to replace that with an array.